crates/core/src/client/mod.rs

```diff
@@ -120,7 +128,7 @@ impl<P: Provider + Send + Sync> KakarotEthApi<P> for KakarotClient<P> {
 
         // Convert the eth log filter to a starknet event filter
         let filter: EthEventFilter = filter.into();
-        let event_filter = filter.to_starknet_event_filter(self)?;
+        let event_filter = filter.to_starknet_filter(self)?;
```

Comment:
You had a problem rebasing, this should be `to_starknet_event_filter`

Reply:
I see, fixed!